src/functionalTest/resources/scenarios/RIA-7332-internal-appellant-ftpa-decided-refused-document.json

```diff
@@ -0,0 +1,48 @@
+{
+  "description": "RIA-7332 Internal appellant ftpa decided - refused letter",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 7332,
+      "eventId": "residentJudgeFtpaDecision",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-internal-appeal-submitted.json",
+        "replacements": {
+          "appellantInDetention": "Yes",
+          "ftpaAppellantRjDecisionOutcomeType": "refused",
+          "ftpaApplicantType": "appellant"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-internal-appeal-submitted.json",
+      "replacements": {
+        "appellantInDetention": "Yes",
+        "ftpaAppellantRjDecisionOutcomeType": "refused",
+        "ftpaApplicantType": "appellant",
+        "notificationAttachmentDocuments": [
+          {
+            "id": "1",
+            "value": {
+              "tag": "internalAppellantFtpaDecidedLetter",
+              "document": {
+                "document_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/",
+                "document_filename": "PA 12345 2019-Awan-appellant-ftpa-decided-refused-letter.PDF",
+                "document_binary_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\/binary/"
+              },
+              "suppliedBy": "",
+              "description": "",
+              "dateUploaded": "{$TODAY}"
+            }
+          }
+        ]
+      }
+    }
+  }
+}

```

Comment:
I think we should have 2 FTs per scenario (granted/partially granted/refused), one for ADA and one for non-ada. Although they both should work as the handler conditions have just isAppellantInDetention, having 2 FTs ensures against future changes.

Reply:
Added